src/main/java/seedu/address/model/activity/Amount.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.model.activity;
+
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents an expense Amount in an Expense.
+ * Guarantees: immutable; is valid as declared in {@link #isValidAmount(double)}
+ */
+public class Amount {
+    public static final String MESSAGE_CONSTRAINTS =
+            "The amount of money spent can only be a positive number";
+
+    public final double value;
```

Comment:
I was thinking, should we use two integers (dollars and cents) instead of using double? Or we should at least force rounding of the double to the nearest 2 decimal places somewhere down the line

Reply:
We are sticking with `double`.